docs/webhook/webhook-samples.mdx

```diff
@@ -8,18 +8,22 @@ import TabItem from '@theme/TabItem';
 
 Toda vez que um pix for realizado a API irá enviar um payload para sua API do Webhook. De acordo com o tipo do Pix será enviado um payload para tal.
```

Comment:
I guess we need to improve here and split each payload by event type

Transaction received
- detached pix payload
- qrcode pix payload
- pix payload

Transaction Refunded
- pix transaction refund payload